cmd/kyma/test/run/cmd.go

```diff
@@ -50,6 +52,7 @@ To execute all test defintions, run ` + "`kyma test run -n example-test`" + `.
 	}
 
 	cobraCmd.Flags().StringVarP(&o.Name, "name", "n", "", `Name of the new test suite. If you don't specify the value for the "-n" flag, the name of the test suite will be autogenerated.`)
+	cobraCmd.Flags().StringArrayVarP(&o.LabelExpressions, "label-expression", "l", []string{}, `label expression used to select the tests for the new test suite.`)
```

Comment:
why not "--labels", the expression part sounds redundant.
Or "--selector" as in kubectl

Reply:
--selector it is

Reply:
besides this it looks good to me

Reply:
fixed